### DISCUSSION TITLE
How do I got the informative HTML file for generated images similar to fooocus?

### DISCUSSION THREAD

**Discussion by skibidiskoobidi** [lllyasviel/stable-diffusion-webui-forge#1808] [Q&A]
I really liked that feature. I only see the option for individual text files. Maybe I'm overlooking the HTML option. Thanks

**Comment by Liva24** [comment]
Do you mean the info/prompts used to generate the image?

**Reply by skibidiskoobidi** [reply]
essentially yes. fooocus produced an HTML file called log.html in the output directory which was automatically updated with all the output image info. it was really handy. 
fooocus and forge are produced by the same person so i was thinking this feature would also be there and i'm probably overlooking a setting somehwere.

**Reply by Liva24** [reply]
oh ok just making sure before I give you my answer... So, all that info is in the png itself, just drag you generated image to PNG INFO Tab and you will see the prompts, sizes, etc. or if you have the image browser extension you can see it from there too.

**Reply by skibidiskoobidi** [reply]
thanks for the tip! works very well with images from civitai made by others but not with mine. apparently i've got mine set to private or something? i guess? 
i would still like the log.html though

**Reply by Liva24** [reply]
rly? is it .png right? by any chance are you upscaling it?
